src/app/common/user.service.ts

```diff
@@ -2,15 +2,26 @@ import { Injectable } from '@angular/core';
 import { BehaviorSubject } from 'rxjs/BehaviorSubject';
 
 import { User } from '../model';
+import { OAuthService } from 'angular-oauth2-oidc-hybrid';
 
 @Injectable()
 export class UserService {
   private _user = new BehaviorSubject(<User>{});
 
+  constructor(private oauthService: OAuthService) {}
+
   get user() {
     return this._user.asObservable();
   }
 
+  /**
+   * Checks to see if user has a token or not
+   */
+  checkToken() {
+    console.log('user.service.ts this.oauthService.hasValidAccessToken(): ' + JSON.stringify(this.oauthService.hasValidAccessToken()));
```

Comment:
Honestly should really just change these `console.log` statements to `log.debug` statements, otherwise we start getting lots of console noise.